### DISCUSSION TITLE
I need lead data

### DISCUSSION THREAD

**Discussion by Ajmal-hassan** [getonbrd/pipedrive-connect#6] [Q&A]
How can i fetch all leads data?

**Comment by j4rs** [comment]
Hello, thanks for reaching out.

I just added added the models **Lead**, **LeadSource**, **LeadLabel** and **Goal** to the recent version **v1.2.0** of the gem. You could simply request all the leads by calling:

```ruby
Pipedrive::Lead.all
```

Please check it out and let me know if it worked or if there are any issues.
